data/migrations/v0.9.0-cleanup.sql

```diff
@@ -1,7 +1,12 @@
--- old version of the POI level calculation without the 'disused' parameter.
+-- drop no longer used functions, supplanted by generated functions
 DROP FUNCTION IF EXISTS mz_calculate_poi_level(
   text, text, text, text, text, text, text, text, text, text, text, text, text,
   text, text, text, text, text, hstore, real);
+DROP FUNCTION IF EXISTS mz_calculate_landuse_min_zoom(text, text, text, text, text, text, text, text, text, text, real);
+DROP FUNCTION IF EXISTS mz_calculate_landuse_kind(text, text, text, text, text, text, text, man_text, text, text);
```

Comment:
I think the regexp replace might have needed an underscore for `man_made` in this and the next 3 lines?